src/lucky_flow/find_element.cr

```diff
@@ -46,7 +40,8 @@ class LuckyFlow::FindElement
     LuckyFlow.settings
   end
 
-  private def matching_elements : Array(Selenium::Element)
+  private def find_matching_elements : Array(Selenium::Element)
+    self.tries += 1
```

Comment:
In my refactor I forgot to increment this number and ended up in an infinite loop. Rather than fixing it and letting someone else do it in the future, I thought it made more sense to have it here.